app/Http/Controllers/Twill/ExperienceSlideController.php

```diff
@@ -6,22 +6,22 @@
 use App\Repositories\ExperienceRepository;
 use App\Repositories\SlideRepository;
 
-class ExperienceSlideController extends \App\Http\Controllers\Twill\ModuleController
+class ExperienceSlideController extends BaseController
 {
-    protected $moduleName = 'experiences.slides';
-    protected $modelName = 'Slide';
-    protected $previewView = 'site.experienceDetail';
+    protected function setUpController(): void
+    {
+        parent::setUpController();
+        $this->enableReorder();
```

Comment:
In Interactive Features the order of the slide records is important information. I don't think you should enable ordering here.

Reply:
The original controller specified `'reorder' => true` in the index options:
https://github.com/art-institute-of-chicago/artic.edu/pull/685/files#diff-693d7c0e38e3803f5c3e5e56bc3fdc2859d64fc4f1224ade315d3e54e080eab8L21

and in production the slides are reorderable:
![Screenshot 2025-02-04 at 4 19 52 PM](https://github.com/user-attachments/assets/8bc5410c-893d-491a-9dcd-f92d602bfe88)

Reply:
Ah, right, I think I'm directingmy comment to the wrong line of code. I'm referring to the ability to _sort_ the records in the table by title asc and desc. That functionality is not in production.

Reply:
I've updated the title field to be non-sortable.